### DISCUSSION TITLE
How to set up Allure Report using GitLab ?

### DISCUSSION THREAD

**Discussion by ShripadGhatigar** [allure-framework/allure2#1696] [Questions & Support]
I am running selenium scripts inside the container and I need to set up the Allure reporting. Is there any Docker image available to call this inside yml file?

**Comment by VadymIakobchuk** [comment]
https://github.com/simple-elf/allure-report-action

**Reply by fcastells** [reply]
That's only for Github, not Gitlab, right? Unless I missed something...

**Comment by ReachHeaven** [comment]
You can try something like this at you gitlab-ci yaml
```
script:
    - wget https://github.com/allure-framework/allurectl/releases/latest/download/allurectl_linux_amd64 -O ./allurectl
    - chmod +x ./allurectl
    - ./allurectl auth login
    - ./allurectl watch --results target/allure-results -- mvn clean test site
```

**Reply by baev** [reply]
BTW this answer is for [Allure TestOps](https://qameta.io/)
